lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -159,16 +172,38 @@ def setup(self):
 
         self.log.debug(f"Output DL3 file: {self.output_file}")
 
+        if self.optimize_gh and self.input_irf:
+            try:
+                QTable.read(self.input_irf, hdu="GH_CUTS")
+            except KeyError:
+                raise ToolConfigurationError(
+                    f"{self.input_irf} does not have GH CUTS HDU, or "
```

Comment:
Why "or", it's the same thing, right?

Reply:
Yes. I just wanted to add an explanation on the GH_CUTS HDU, as it is not defined by GADF.